CMakeLists.txt

```diff
@@ -151,12 +154,26 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/examples/CMakeLists.txt")
 endif()
 
 
-install(TARGETS ${XMP_INSTALLS}
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "lib"
-        ARCHIVE DESTINATION "lib"
-        INCLUDES DESTINATION "include")
+# === Install ====
+include(GNUInstallDirs)
 
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(exec_prefix "\${prefix}")
+set(includedir "${CMAKE_INSTALL_FULL_INCLUDEDIR}")
+set(libdir     "${CMAKE_INSTALL_FULL_LIBDIR}")
+set(bindir     "${CMAKE_INSTALL_FULL_BINDIR}")
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/libxmp.pc.in"
+               "${CMAKE_CURRENT_BINARY_DIR}/libxmp.pc" @ONLY
+)
+install(TARGETS ${XMP_INSTALLS}
```

Comment:
I'd added the space between `configure_file()` and `install()` here to make the thing look nicer, however, fine.